servo-core/src/main/java/com/netflix/servo/monitor/ContextualCounter.java

```diff
@@ -15,9 +15,10 @@
  */
 package com.netflix.servo.monitor;
 
-import com.google.common.base.Function;
 import com.netflix.servo.tag.TaggingContext;
 
+import java.util.function.Function;
```

Comment:
The issue with this is that `Function` is part of the public API for this class. See previous discussions in #288 and #290. We would need to make sure it wouldn't break anything internally. Since servo is deprecated, we would probably focus more on moving users off instead.

Reply:
Ah good point, I can look at fixing the breakages if they occur internally.  I also took a quick look at who is still using servo and it's a bit intertwined into internal platform libraries which are also deprecated.  I can also recommend full removal of servo but figured this would get some benefit in the interim.